# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# dcantrel <dcantrell@redhat.com>, 2011.
# Dimitris Glezos <glezos@indifex.com>, 2011.
# Miguel Sousa <migueljorgesousa@sapo.pt>, 2012.
# Ricardo Pinto <ricardo.bigote@gmail.com>, 2011.
# Rui Gouveia <rui.gouveia@gmail.com>, 2011-2012.
msgid ""
msgstr ""
"Project-Id-Version: Anaconda\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2015-05-19 12:36-0400\n"
"PO-Revision-Date: 2013-01-04 23:55+0000\n"
"Last-Translator: Brian Lane <bcl@redhat.com>\n"
"Language-Team: Portuguese <trans-pt@lists.fedoraproject.org>\n"
"Language: pt\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: pyanaconda/bootloader.py:248
#, fuzzy
msgid "/boot file system"
msgstr "Sistema de ficheiros /boot"

#: pyanaconda/bootloader.py:405
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr ""
"Conjuntos RAID que contêm '%s' devem ter um dos seguintes níveis de raid: %s."

#: pyanaconda/bootloader.py:413
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""
"Conjuntos RAID que contêm '%s' devem ter uma das seguintes versões de "
"metadados: %s."

#: pyanaconda/bootloader.py:421
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr ""
"Conjuntos RAID que contêm '%s' devem ter um dos seguintes tipos de "
"dispositivo: %s."

#: pyanaconda/bootloader.py:437
#, fuzzy, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr "%s tem de ter um dos seguintes tipos de etiqueta de disco: %s."

#: pyanaconda/bootloader.py:449
#, fuzzy, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr "%s não pode ser do tipo %s."

#: pyanaconda/bootloader.py:455
#, fuzzy, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr "%s deve ser montado num dos %s."

#: pyanaconda/bootloader.py:467
#, fuzzy, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr "%s deve estar entre %d e %d MB de tamanho"

#: pyanaconda/bootloader.py:473
#, fuzzy, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr "%s não deve ser menor que %dMB."

#: pyanaconda/bootloader.py:482
#, fuzzy, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr "%s não deve ser maior que %dMB."

#: pyanaconda/bootloader.py:500
#, fuzzy, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr "%s deve estar dentro dos primeiros %dMB do disco."

#: pyanaconda/bootloader.py:510
#, python-format
msgid "%s must be on a primary partition."
msgstr "%s tem de estar numa partição primária."

#: pyanaconda/bootloader.py:646 pyanaconda/bootloader.py:746
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr "%s não pode estar num dispositivo cifrado."

#: pyanaconda/bootloader.py:701
#, python-format
msgid "%s cannot be on an iSCSI disk on s390(x)"
msgstr ""

#: pyanaconda/bootloader.py:705
#, fuzzy, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr "%s não pode ser do tipo %s"

#: pyanaconda/bootloader.py:1375
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not. A drive failure in %(stage2dev)s "
"could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader.py:1381
#, python-format
msgid ""
"boot loader stage2 device %(stage2dev)s is on a multi-disk array, but boot "
"loader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"boot loader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader.py:1669
#, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""

#: pyanaconda/constants.py:73
msgid "Start VNC"
msgstr "Iniciar o VNC"

#: pyanaconda/constants.py:74
msgid "Use text mode"
msgstr "Usar o modo de texto"

#: pyanaconda/constants.py:138
#, fuzzy
msgid "The password is empty."
msgstr "Senha está definida."

#: pyanaconda/constants.py:139
#, fuzzy
msgid "The passwords do not match."
msgstr "As senhas não coincidem!"

#: pyanaconda/constants.py:140
msgid "The passwords you entered were different.  Please try again."
msgstr ""

#: pyanaconda/constants.py:141
#, python-format
msgid "The password you have provided is weak. %s"
msgstr ""

#: pyanaconda/constants.py:142
#, python-format
msgid "The password you have provided is weak: %s. %s"
msgstr ""

#: pyanaconda/constants.py:143
msgid "You have provided a weak password. Press Done again to use anyway."
msgstr ""

#: pyanaconda/constants.py:144
#, python-format
msgid "You have provided a weak password: %s. Press Done again to use anyway."
msgstr ""

#: pyanaconda/constants.py:145
msgid ""
"The password you have provided contains non-ASCII characters. You may not be "
"able to switch between keyboard layouts to login. Press Done to continue."
msgstr ""

#: pyanaconda/constants.py:146
msgid "You will have to press Done twice to confirm it."
msgstr ""

#: pyanaconda/constants.py:148
msgid "Empty"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:125
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:177
msgid "Weak"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:128
msgid "Fair"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:131
msgid "Good"
msgstr ""

#: pyanaconda/constants.py:148 pyanaconda/ui/gui/spokes/lib/passphrase.py:134
msgid "Strong"
msgstr ""

#: pyanaconda/constants_text.py:40 pyanaconda/rescue.py:333
#: pyanaconda/rescue.py:365 pyanaconda/rescue.py:433 pyanaconda/rescue.py:449
#: pyanaconda/rescue.py:460 anaconda:513
msgid "OK"
msgstr "OK"

#: pyanaconda/constants_text.py:44 pyanaconda/text.py:52 pyanaconda/text.py:53
#, fuzzy
msgid "Cancel"
msgstr "_Cancelar"

#: pyanaconda/constants_text.py:48
msgid "Yes"
msgstr ""

#: pyanaconda/constants_text.py:52
msgid "No"
msgstr ""

#: pyanaconda/errors.py:109
#, fuzzy, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""
"O seguinte erro ocorreu durante a configuração do repositório de "
"instalação:\n"
"\n"
"%(e)s\n"
"\n"
"a instalação não pode continuar."

#: pyanaconda/errors.py:115
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr ""

#: pyanaconda/errors.py:124
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/errors.py:133
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect file system type:\n"
"\n"
msgstr ""

#: pyanaconda/errors.py:139
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""

#: pyanaconda/errors.py:153
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

#: pyanaconda/errors.py:163
#, python-format
msgid "An error occurred mounting the source device %s. Retry?"
msgstr ""

#: pyanaconda/errors.py:171
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: pyanaconda/errors.py:179
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist. "
"This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:185
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:190
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:202
#, python-format
msgid ""
"The package '%s' is required for this installation. This package does not "
"exist. This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:209
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:220
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:228
#, fuzzy
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""
"O seguinte erro ocorreu durante a configuração do repositório de "
"instalação:\n"
"\n"
"%(e)s\n"
"\n"
"a instalação não pode continuar."

#: pyanaconda/errors.py:236
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source."
msgstr ""

#: pyanaconda/errors.py:244
#, fuzzy
msgid ""
"The following error occurred while installing the boot loader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""
"O seguinte erro ocorreu durante a configuração do repositório de "
"instalação:\n"
"\n"
"%(e)s\n"
"\n"
"a instalação não pode continuar."

#: pyanaconda/errors.py:256
#, python-format
msgid "Unable to encrypt password: unsupported algorithm %s"
msgstr ""

#: pyanaconda/errors.py:263
#, fuzzy, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The "
"exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""
"A instalação foi interrompida devido ao que parece ser um problema com o "
"hardware. A mensagem de erro exacta é:\n"
"\n"
"%s.\n"
"\n"
" O instalador irá agora terminar."

#: pyanaconda/exception.py:79
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""
"A instalação foi interrompida devido ao que parece ser um problema com o "
"hardware. A mensagem de erro exacta é:\n"
"\n"
"%s.\n"
"\n"
" O instalador irá agora terminar."

#: pyanaconda/exception.py:83
#, fuzzy
msgid "Hardware error occured"
msgstr "Erro de hardware encontrado"

#: pyanaconda/exception.py:136
#, fuzzy, python-format
msgid ""
"\n"
"The installation was stopped due to incomplete spokes detected while running "
"in non-interactive cmdline mode. Since there cannot be any questions in "
"cmdline mode, edit your kickstart file and retry installation.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""
"A instalação foi interrompida devido ao que parece ser um problema com o "
"hardware. A mensagem de erro exacta é:\n"
"\n"
"%s.\n"
"\n"
" O instalador irá agora terminar."

#: pyanaconda/exception.py:144
#, fuzzy, python-format
msgid ""
"\n"
"Running in cmdline mode, no interactive debugging allowed.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""
"A instalação foi interrompida devido ao que parece ser um problema com o "
"hardware. A mensagem de erro exacta é:\n"
"\n"
"%s.\n"
"\n"
" O instalador irá agora terminar."

#: pyanaconda/install.py:80
msgid "Configuring installed system"
msgstr ""

#: pyanaconda/install.py:93
msgid "Writing network configuration"
msgstr ""

#: pyanaconda/install.py:97
msgid "Creating users"
msgstr ""

#: pyanaconda/install.py:105
msgid "Configuring addons"
msgstr ""

#: pyanaconda/install.py:108
msgid "Generating initramfs"
msgstr ""

#: pyanaconda/install.py:119
#, python-format
msgid "Joining realm: %s"
msgstr ""

#: pyanaconda/install.py:122
#, fuzzy
msgid "Running post-installation scripts"
msgstr "A correr os scripts pré-instalação"

#: pyanaconda/install.py:170
#, python-format
msgid "Waiting for %s threads to finish"
msgstr ""

#: pyanaconda/install.py:176
#, fuzzy
msgid "Setting up the installation environment"
msgstr "A iniciar processo de instalação de pacotes"

#: pyanaconda/install.py:207
msgid "Discovering realm to join"
msgstr ""

#: pyanaconda/install.py:265
#, fuzzy
msgid "Installing boot loader"
msgstr "A instalar software"

#: pyanaconda/install.py:268 pyanaconda/packaging/livepayload.py:176
#: pyanaconda/packaging/yumpayload.py:1371
#: pyanaconda/packaging/dnfpayload.py:667
#, fuzzy
msgid "Performing post-installation setup tasks"
msgstr "A executar configuração pós-instalação"

#: pyanaconda/iutil.py:811
msgid "No host url"
msgstr ""

#: pyanaconda/iutil.py:832
msgid "malformed URL, cannot parse it."
msgstr ""

#: pyanaconda/iutil.py:848
msgid "URL has no host component"
msgstr ""

#: pyanaconda/kickstart.py:164
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr ""

#: pyanaconda/kickstart.py:172
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"Foi encontrado o seguinte erro ao transferir o certificado:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:281
#, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr ""

#: pyanaconda/kickstart.py:306
#, python-format
msgid "autopart fstype of %s is invalid."
msgstr ""

#: pyanaconda/kickstart.py:323
#, python-format
msgid "Settings default fstype to %s failed."
msgstr ""

#: pyanaconda/kickstart.py:354
msgid "GRUB2 does not support installation to a partition."
msgstr ""

#: pyanaconda/kickstart.py:414
#, python-format
msgid "More than one match found for given boot drive \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:417
#, python-format
msgid "Requested boot drive \"%s\" doesn't exist or cannot be used."
msgstr ""

#: pyanaconda/kickstart.py:459
#, python-format
msgid ""
"Btrfs partition \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"btrfs\"."
msgstr ""

#: pyanaconda/kickstart.py:464
#, python-format
msgid "Tried to use undefined partition \"%s\" in Btrfs volume specification."
msgstr ""

#: pyanaconda/kickstart.py:477
msgid ""
"Btrfs volume defined without any member devices.  Either specify member "
"devices or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:486 pyanaconda/kickstart.py:859
#: pyanaconda/kickstart.py:1438
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr ""

#: pyanaconda/kickstart.py:501
#, python-format
msgid "Btrfs volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:587
#, python-format
msgid ""
"Disklabel \"%s\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/kickstart.py:599
#, python-format
msgid "Disk \"%s\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:612
#, python-format
msgid "Device \"%s\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/kickstart.py:639
#, python-format
msgid "NIC \"%s\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:690
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr ""

#: pyanaconda/kickstart.py:738 pyanaconda/kickstart.py:749
#, python-format
msgid "Disk \"%s\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:762
#, python-format
msgid ""
"Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target "
"is not up."
msgstr ""

#: pyanaconda/kickstart.py:772
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for "
"none"
msgstr ""

#: pyanaconda/kickstart.py:865
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/kickstart.py:872
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/kickstart.py:880
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/kickstart.py:885 pyanaconda/kickstart.py:946
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:896 pyanaconda/kickstart.py:905
#: pyanaconda/kickstart.py:956 pyanaconda/kickstart.py:1215
#: pyanaconda/kickstart.py:1224 pyanaconda/kickstart.py:1306
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:919
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/kickstart.py:924
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/kickstart.py:935 pyanaconda/kickstart.py:1242
#: pyanaconda/kickstart.py:1488
#, python-format
msgid "The \"%s\" file system type is not supported."
msgstr ""

#: pyanaconda/kickstart.py:1029
msgid "No passphrase given for encrypted LV"
msgstr ""

#: pyanaconda/kickstart.py:1083 pyanaconda/kickstart.py:1087
#, python-format
msgid "The %s kickstart command is not currently supported."
msgstr ""

#: pyanaconda/kickstart.py:1112
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1147
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1158 pyanaconda/kickstart.py:1413
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1169 pyanaconda/kickstart.py:1423
#, python-format
msgid "Btrfs partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1192
#, python-format
msgid "The size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1199
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/kickstart.py:1204 pyanaconda/kickstart.py:1297
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1257 pyanaconda/kickstart.py:1271
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1260
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1267
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/kickstart.py:1280
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1350
msgid "No passphrase given for encrypted part"
msgstr ""

#: pyanaconda/kickstart.py:1445
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/kickstart.py:1450
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1471
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a "
"format of \"mdmember\"."
msgstr ""

#: pyanaconda/kickstart.py:1476
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/kickstart.py:1504
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1511
#, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:1726
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/kickstart.py:1731
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/kickstart.py:1737
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:1747
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be "
"one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/kickstart.py:1754
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""

#: pyanaconda/kickstart.py:1759
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1762
#, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr ""

#: pyanaconda/kickstart.py:1818
msgid ""
"The upgrade kickstart command is no longer supported. Upgrade functionality "
"is provided through fedup."
msgstr ""

#: pyanaconda/kickstart.py:2092
msgid "Running pre-installation scripts"
msgstr "A correr os scripts pré-instalação"

#: pyanaconda/network.py:98
msgid "Host name cannot be None or an empty string."
msgstr ""

#: pyanaconda/network.py:101
#, fuzzy
msgid "Host name must be 255 or fewer characters in length."
msgstr "O nome da máquina tem de ter 255 caracteres ou menos de tamanho."

#: pyanaconda/network.py:104
msgid ""
"Host names can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""

#: pyanaconda/network.py:1325 pyanaconda/ui/gui/spokes/custom.py:545
#: pyanaconda/ui/gui/spokes/network.py:302
#: pyanaconda/ui/gui/spokes/lib/accordion.py:61
msgid "Unknown"
msgstr ""

#: pyanaconda/network.py:1329
msgid "Connecting..."
msgstr ""

#: pyanaconda/network.py:1331
msgid "Disconnecting..."
msgstr ""

#: pyanaconda/network.py:1351
#, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr ""

#: pyanaconda/network.py:1354
#, python-format
msgid "Wireless connected to %(access_point)s"
msgstr ""

#: pyanaconda/network.py:1357
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1361
#, python-format
msgid "Team%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1365
#, python-format
msgid "Bridge%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1371
#, python-format
msgid "VLAN %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1388
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr ""

#: pyanaconda/network.py:1391
msgid "Not connected"
msgstr ""

#: pyanaconda/network.py:1394 pyanaconda/ui/gui/spokes/network.glade:2097
msgid "No network devices available"
msgstr ""

#: pyanaconda/product.py:53
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr ""

#: pyanaconda/product.py:57
#, fuzzy, python-format
msgid "New %(name)s %(version)s Installation"
msgstr "Instalação do %(productName)s %(productVersion)s"

#: pyanaconda/rescue.py:184 pyanaconda/rescue.py:350 pyanaconda/rescue.py:426
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr "Execute %s para desmontar o sistema quando tiver terminado."

#: pyanaconda/rescue.py:187
msgid "When finished please exit from the shell and your system will reboot."
msgstr "Quando terminar saia da consola e o seu sistema será reiniciado."

#: pyanaconda/rescue.py:200
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""
"Não foi possível encontrar /bin/sh para executar!  Consola não iniciada"

#: pyanaconda/rescue.py:288 pyanaconda/rescue.py:358 pyanaconda/rescue.py:429
#: pyanaconda/rescue.py:460
msgid "Rescue"
msgstr "Recuperação"

#: pyanaconda/rescue.py:289
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  \n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:299
msgid "Continue"
msgstr "Continuar"

#: pyanaconda/rescue.py:299 pyanaconda/rescue.py:305
msgid "Read-Only"
msgstr "Apenas para Leitura"

#: pyanaconda/rescue.py:299 pyanaconda/rescue.py:301
msgid "Skip"
msgstr "Saltar"

#: pyanaconda/rescue.py:330
msgid "System to Rescue"
msgstr "Sistema a Recuperar"

#: pyanaconda/rescue.py:331
msgid "Which device holds the root partition of your installation?"
msgstr "Que dispositivo tem a partição root da sua instalação?"

#: pyanaconda/rescue.py:333 pyanaconda/rescue.py:337
msgid "Exit"
msgstr "Sair"

#: pyanaconda/rescue.py:347 pyanaconda/rescue.py:423
msgid "The system will reboot automatically when you exit from the shell."
msgstr "O sistema será reiniciado automaticamente quando sair da consola."

#: pyanaconda/rescue.py:359
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"
msgstr ""
"O sistema foi montado em %(rootPath)s.\n"
"\n"
"Pressione em <enter> para obter uma consola. Se pretende fazer do seu "
"sistema o ambiente root, execute o comando:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"

#: pyanaconda/rescue.py:430
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell."
msgstr ""
"Um erro ocorreu ao tentar montar parte ou a totalidade de seu sistema. Uma "
"parte poderá estar montada em %s.\n"
"\n"
"Pressione <return> para obter uma consola."

#: pyanaconda/rescue.py:438
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Você não tem nenhuma partição Linux.  A reiniciar.\n"

#: pyanaconda/rescue.py:442
msgid " The system will reboot automatically when you exit from the shell."
msgstr " O sistema será reiniciado automaticamente quando sair da consola."

#: pyanaconda/rescue.py:446
msgid "Rescue Mode"
msgstr "Modo de Recuperação"

#: pyanaconda/rescue.py:447
#, python-format
msgid "You don't have any Linux partitions. Press return to get a shell.%s"
msgstr ""
"Não tem nenhuma partição Linux. Pressione Enter para obter uma consola.%s"

#: pyanaconda/rescue.py:459
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "O seu sistema está montado na directoria %s."

#: pyanaconda/storage_utils.py:50 pyanaconda/storage_utils.py:82
#: pyanaconda/ui/gui/spokes/custom.glade:31
msgid "LVM"
msgstr ""

#: pyanaconda/storage_utils.py:51 pyanaconda/storage_utils.py:83
#: pyanaconda/ui/gui/spokes/custom.glade:39
msgid "LVM Thin Provisioning"
msgstr ""

#: pyanaconda/storage_utils.py:52 pyanaconda/ui/gui/spokes/custom.glade:35
msgid "RAID"
msgstr ""

#: pyanaconda/storage_utils.py:53 pyanaconda/storage_utils.py:80
#: pyanaconda/ui/gui/spokes/custom.glade:23
msgid "Standard Partition"
msgstr ""

#: pyanaconda/storage_utils.py:54 pyanaconda/storage_utils.py:81
#: pyanaconda/ui/gui/spokes/custom.glade:27
msgid "Btrfs"
msgstr ""

#: pyanaconda/storage_utils.py:55
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:713
#: pyanaconda/ui/gui/spokes/lib/resize.glade:152
msgid "Disk"
msgstr ""

#: pyanaconda/storage_utils.py:66
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""

#: pyanaconda/storage_utils.py:68
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""

#: pyanaconda/storage_utils.py:70
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""

#: pyanaconda/storage_utils.py:72
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""

#: pyanaconda/storage_utils.py:74
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""

#: pyanaconda/storage_utils.py:76
msgid ""
"The PReP boot partition is required as part of the\n"
"boot loader configuration on some PPC platforms."
msgstr ""

#: pyanaconda/storage_utils.py:184
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"A sua partição raiz tem menos do que 250 megabytes, o que significa que "
"provavelmente é demasiado pequena para instalar o %s."

#: pyanaconda/storage_utils.py:189
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"Não definiu um partição raiz (/), que é necessária para a instalação do %s "
"continuar."

#: pyanaconda/storage_utils.py:203
msgid ""
"This platform requires /boot on a dedicated partition or logical volume.  If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""
"Esta plataforma requer que o /boot esteja numa partição dedicada ou volume "
"lógico.  Se não quer um volume /boot, deve colocar a / numa partição "
"dedicada não-LVM."

#: pyanaconda/storage_utils.py:213
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""
"A partição %(mount)s tem menos do que %(size)s MB, o que é inferior ao "
"recomendado para uma instalação normal de %(productName)s."

#: pyanaconda/storage_utils.py:223
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""
"A partição %(mount)s é muito pequena para uma formatação %(format)s (tamanho "
"permitido vai de %(minSize)d MB a %(maxSize)d MB)"

#: pyanaconda/storage_utils.py:229
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""
"A partição %(mount)s é grande demais para uma formatação %(format)s (tamanho "
"permitido vai de %(minSize)d MB a %(maxSize)d MB)"

#: pyanaconda/storage_utils.py:238
msgid "No valid boot loader target device found. See below for details."
msgstr ""

#: pyanaconda/storage_utils.py:250
msgid "You have not created a bootable partition."
msgstr "Ainda não criou uma partição de arranque."

#: pyanaconda/storage_utils.py:272
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""

#: pyanaconda/storage_utils.py:283
#, fuzzy, python-format
msgid ""
"You have not specified a swap partition.  %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""
"Não especificou uma partição swap.  %(requiredMem)s MB de memória são "
"necessários para continuar a instalação sem uma partição swap, mas o sistema "
"tem apenas %(installedMem)s MB."

#: pyanaconda/storage_utils.py:290
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"Não indicou um partição swap.  Embora não seja estritamente obrigatório, a "
"sua utilização aumenta significativamente a performance na maior parte das "
"instalações."

#: pyanaconda/storage_utils.py:297
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""
"Pelo menos um dos dispositivos de swap não tem um UUID, o que é comum no "
"espaço de swap criado com versões antigas do mkswap. Estes dispositivos vão "
"ser referenciados pelo caminho do dispositivo no /etc/fstab, o que não é "
"ideal, pois os caminhos dos dispositivos podem mudar sob várias "
"circunstâncias. "

#: pyanaconda/storage_utils.py:308
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""
"Este ponto de montagem é inválido.  A directoria %s deve estar no sistema de "
"ficheiros /."

#: pyanaconda/storage_utils.py:313
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "O ponto de montagem %s deve estar num sistema de ficheiros linux."

#: pyanaconda/storage_utils.py:341
#, python-format
msgid ""
"Encryption requested for LUKS device %s but no encryption key specified for "
"this device."
msgstr ""

#: pyanaconda/text.py:99
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: pyanaconda/text.py:106
#, fuzzy
msgid "Passphrase"
msgstr "Senha está definida."

#: pyanaconda/vnc.py:74
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "Instalação do %(productName)s %(productVersion)s"

#: pyanaconda/vnc.py:135
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr "Instalação do %(productName)s %(productVersion)s na máquina %(name)s"

#: pyanaconda/vnc.py:154
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "A tentar ligar o cliente vnc ao computador %s..."

#: pyanaconda/vnc.py:168
msgid "Connected!"
msgstr "Ligado!"

#: pyanaconda/vnc.py:171
msgid "Will try to connect again in 15 seconds..."
msgstr "Irá tentar ligar-se de novo daqui a 15 segundos..."

#: pyanaconda/vnc.py:178
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "A desistir da ligação ao fim de %d tentativa!\n"
msgstr[1] "A desistir da ligação ao fim de %d tentativas!\n"

#: pyanaconda/vnc.py:186
msgid "Attempting to start vncconfig"
msgstr ""

#: pyanaconda/vnc.py:199
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr "Ligue manualmente o seu cliente vnc a %s para iniciar a instalação."

#: pyanaconda/vnc.py:201
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""

#: pyanaconda/vnc.py:206
msgid "Starting VNC..."
msgstr "A iniciar o VNC..."

#: pyanaconda/vnc.py:243
msgid "The VNC server is now running."
msgstr "O servidor de VNC está a correr."

#: pyanaconda/vnc.py:247
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Optou por ligar-se a um vncviewer à escuta. \n"
"Isto não requer a definição de uma senha.  Se a definir, \n"
"ela será usada no caso da ligação ao vncviewer não \n"
"ter sucesso\n"
"\n"

#: pyanaconda/vnc.py:252
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"ATENÇÃO!!! O servidor de VNC está a correr SEM SENHA!\n"
"Você pode usar a opção de arranque vncpassword=<senha>\n"
"se quiser tornar o servidor seguro.\n"
"\n"

#: pyanaconda/vnc.py:256
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"Optou por executar o vnc com uma senha. \n"
"\n"

#: pyanaconda/vnc.py:258
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"Erro Desconhecido.  A interromper. \n"
"\n"

#: pyanaconda/vnc.py:280
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""

#: pyanaconda/packaging/__init__.py:1113
#, fuzzy
msgid "Failed to set up installation source"
msgstr "A preparar transacção a partir das fontes de instalação"

#: pyanaconda/packaging/__init__.py:1114
#, fuzzy
msgid "Error downloading package metadata"
msgstr "Erro ao instalar pacote"

#: pyanaconda/packaging/__init__.py:1115
msgid "No installation source available"
msgstr ""

#: pyanaconda/packaging/livepayload.py:104
#: pyanaconda/packaging/livepayload.py:125
msgid "Installing software"
msgstr "A instalar software"

#: pyanaconda/packaging/livepayload.py:237
#: pyanaconda/packaging/livepayload.py:246
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr ""

#: pyanaconda/packaging/livepayload.py:375
msgid "Checking image checksum"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1186
#, python-format
msgid "CmdlineError: Missing package: %s"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1331
#: pyanaconda/packaging/dnfpayload.py:621
msgid "Starting package installation process"
msgstr "A iniciar processo de instalação de pacotes"

#: pyanaconda/packaging/yumpayload.py:1369
#: pyanaconda/packaging/dnfpayload.py:650
msgid "Preparing transaction from installation source"
msgstr "A preparar transacção a partir das fontes de instalação"

#: pyanaconda/packaging/yumpayload.py:1370
msgid "Installing"
msgstr "A instalar"

#: pyanaconda/packaging/dnfpayload.py:146
#, python-format
msgid ""
"Downloading %(total_files)s RPMs, %(downloaded)s / %(total_size)s "
"(%(percent)d%%) done."
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:638
#, fuzzy
msgid "Downloading packages"
msgstr "Erro ao instalar pacote"

#: pyanaconda/packaging/dnfpayload.py:660
#, fuzzy, python-format
msgid "Installing %s"
msgstr "A instalar"

#: pyanaconda/packaging/rpmostreepayload.py:163
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:179
#, fuzzy, python-format
msgid "Preparing deployment of %s"
msgstr "A preparar para instalar"

#: pyanaconda/packaging/rpmostreepayload.py:199
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:202
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/ui/helpers.py:94 pyanaconda/ui/tui/spokes/storage.py:389
msgid "Checking storage configuration..."
msgstr ""

#: pyanaconda/ui/categories/__init__.py:48
msgid "DEFAULT TITLE"
msgstr ""

#: pyanaconda/ui/categories/customization.py:32
msgid "CUSTOMIZATION"
msgstr ""

#: pyanaconda/ui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr ""

#: pyanaconda/ui/categories/software.py:31
msgid "SOFTWARE"
msgstr ""

#: pyanaconda/ui/categories/system.py:32
#: pyanaconda/ui/gui/spokes/lib/accordion.py:166
msgid "SYSTEM"
msgstr ""

#: pyanaconda/ui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr ""

#: pyanaconda/ui/lib/space.py:37
#, python-format
msgid ""
"Not enough space in file systems for the current software selection. An "
"additional %s is needed."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:58
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:68
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:92
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:95
#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:88
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/lib/entropy.py:102
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:107
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:110
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:35
#, fuzzy
msgid "Installation"
msgstr "Pós Instalação"

#: pyanaconda/ui/tui/hubs/summary.py:51
msgid "Starting automated install"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:85
msgid "The following mandatory spokes are not completed:"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:93
msgid ""
"  Please make your choice from above ['q' to quit | 'b' to begin "
"installation |\n"
"  'r' to refresh]: "
msgstr ""

#. TRANSLATORS: 'b' to begin installation
#: pyanaconda/ui/tui/hubs/summary.py:107
msgctxt "TUI|Spoke Navigation"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:110 pyanaconda/ui/tui/hubs/__init__.py:114
msgid "Please complete all spokes before continuing"
msgstr ""

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:121 pyanaconda/ui/tui/hubs/__init__.py:111
#: pyanaconda/ui/tui/simpleline/base.py:456
#: pyanaconda/ui/tui/spokes/software.py:189
#, fuzzy
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr "c"

#: pyanaconda/ui/tui/hubs/__init__.py:41
msgid "Default HUB title"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:30 pyanaconda/ui/gui/main.glade:72
msgid "Error"
msgstr "Erro"

#: pyanaconda/ui/tui/tuiobject.py:51
msgid "Press enter to exit."
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:61
msgid "Question"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:84
msgid "Please respond 'yes' or 'no': "
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:87
msgid "yes"
msgstr ""

#: pyanaconda/ui/tui/tuiobject.py:92
msgid "no"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:87
msgid "Do you really want to quit?"
msgstr "Deseja realmente sair?"

#. TRANSLATORS: 'r' to refresh
#: pyanaconda/ui/tui/simpleline/base.py:450
msgctxt "TUI|Spoke Navigation"
msgid "r"
msgstr ""

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/simpleline/base.py:462
#, fuzzy
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/simpleline/base.py:571
msgid "Press ENTER to continue"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:621
msgid ""
"  Please make your choice from above ['q' to quit | 'c' to continue |\n"
"  'r' to refresh]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:42
msgid "VNC"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:58
#, fuzzy
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""
"Não foi possível iniciar o X no seu computador.  Deseja iniciar o VNC para "
"se ligar a este computador a partir de outro computador e realizar uma "
"instalação gráfica ou continuar com uma instalação em modo texto?"

#: pyanaconda/ui/tui/spokes/askvnc.py:106
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/spokes/askvnc.py:122
msgid "VNC Password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:130
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""

#: pyanaconda/ui/tui/spokes/askvnc.py:149
#: pyanaconda/ui/gui/spokes/network.py:1319
msgid "Password: "
msgstr "Senha: "

#: pyanaconda/ui/tui/spokes/askvnc.py:150
msgid "Password (confirm): "
msgstr "Senha (confirmar): "

#: pyanaconda/ui/tui/spokes/askvnc.py:153
msgid "Passwords do not match!"
msgstr "As senhas não coincidem!"

#: pyanaconda/ui/tui/spokes/askvnc.py:156
msgid "The password must be at least six characters long."
msgstr "A senha deve ter pelo menos seis caracteres."

#: pyanaconda/ui/tui/spokes/askvnc.py:160
#, fuzzy
msgid "The password cannot be more than eight characters long."
msgstr "A senha deve ter pelo menos seis caracteres."

#: pyanaconda/ui/tui/spokes/langsupport.py:38
msgid "Language settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:65
msgid "Language is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:75
msgid "Available locales"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:78
msgid "Available languages"
msgstr ""

#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/spokes/langsupport.py:112
msgctxt "TUI|Spoke Navigation|Language Support"
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/spokes/langsupport.py:120
msgid ""
"Please select language support to install.\n"
"[b to language list, c to continue, q to quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:43
msgid "Network configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:94
#, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:99
#, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:111
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:113
#, python-format
msgid " DNS: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:120
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:140
#, fuzzy, python-format
msgid "Host Name: %s\n"
msgstr "Nome de utilizador"

#: pyanaconda/ui/tui/spokes/network.py:152
#, fuzzy
msgid "Set host name"
msgstr "Nome de utilizador"

#: pyanaconda/ui/tui/spokes/network.py:154
#, python-format
msgid "Configure device %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:173
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:241
#: pyanaconda/ui/gui/spokes/network.glade:2233
#, fuzzy
msgid "Host Name"
msgstr "Nome de utilizador"

#: pyanaconda/ui/tui/spokes/network.py:191
msgid ""
"Configuration not saved: gateway or netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:207
msgid "Can't apply configuration, device activation failed."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:234
#: pyanaconda/ui/gui/spokes/network.py:1466
#: pyanaconda/ui/gui/spokes/network.py:1535
#, python-format
msgid "Host name is not valid: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:258
msgid "Device configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:262
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:264
msgid "IPv4 netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:265
msgid "IPv4 gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:266
#, python-format
msgid ""
"IPv6 address or %(auto)s for automatic, %(dhcp)s for DHCP, %(ignore)s to "
"turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:269
msgid "IPv6 default gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:270
msgid "Nameservers (comma separated)"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:271
msgid "Connect automatically after reboot"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:272
#, fuzzy
msgid "Apply configuration in installer"
msgstr "falta um tipo na configuração do sistema de ficheiros"

#: pyanaconda/ui/tui/spokes/network.py:287
#, python-format
msgid "Configuring device %s."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:32
#, fuzzy
msgid "Root password"
msgstr "Definir senha de root"

#: pyanaconda/ui/tui/spokes/password.py:55
msgid "Password is set."
msgstr "Senha está definida."

#: pyanaconda/ui/tui/spokes/password.py:57
msgid "Root account is disabled."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:59
msgid "Password is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:64
msgid "Please select new root password. You will have to type it twice."
msgstr ""

#: pyanaconda/ui/tui/spokes/password.py:70
#: pyanaconda/ui/gui/spokes/password.glade:86
#: pyanaconda/ui/gui/spokes/user.glade:170
#, fuzzy
msgid "Password"
msgstr "Senha: "

#: pyanaconda/ui/tui/spokes/storage.py:55
msgid "Use All Space"
msgstr "Utilizar todo o espaço"

#: pyanaconda/ui/tui/spokes/storage.py:56
msgid "Replace Existing Linux system(s)"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:57
msgid "Use Free Space"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:67
#, fuzzy
msgid "Installation Destination"
msgstr "A Iniciar a Instalação"

#: pyanaconda/ui/tui/spokes/storage.py:123
#: pyanaconda/ui/gui/spokes/storage.py:380
msgid "No disks selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:126
#: pyanaconda/ui/gui/spokes/storage.py:383
msgid "Kickstart insufficient"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:133
#: pyanaconda/ui/gui/spokes/storage.py:392
msgid "Error checking storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:135
#: pyanaconda/ui/gui/spokes/storage.py:394
#, fuzzy
msgid "Warning checking storage configuration"
msgstr "A efectuar configuração pós-actualização"

#: pyanaconda/ui/tui/spokes/storage.py:138
#: pyanaconda/ui/gui/spokes/storage.py:396
msgid "Automatic partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:140
#: pyanaconda/ui/gui/spokes/storage.py:398
msgid "Custom partitioning selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:176
#: pyanaconda/ui/gui/spokes/storage.py:649
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:178
#: pyanaconda/ui/gui/spokes/storage.py:651
msgid "No disks selected; please select at least one disk to install to."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:193
#: pyanaconda/ui/tui/spokes/source.py:366
#: pyanaconda/ui/gui/spokes/source.py:657
#: pyanaconda/ui/gui/spokes/storage.py:600
msgid "Probing storage..."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:214
msgid "Select all"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:307
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:309
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:324
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.py:92
#, python-format
msgid "Formatting /dev/%s. This may take a moment."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:368
msgid "Generating updated storage configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:373
#: pyanaconda/ui/tui/spokes/storage.py:385
#, python-format
msgid "storage configuration failed: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:432
msgid "Autopartitioning Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:458
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:497
msgid "Partition Scheme Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:521
msgid "Select a partition scheme configuration."
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:31
msgid "Timezone settings"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:61
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:533
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:539
#, python-format
msgid "%s timezone"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:63
msgid "Timezone is not set."
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:71
#, python-format
msgid "Available timezones in region %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:74
msgid "Available regions"
msgstr ""

#: pyanaconda/ui/tui/spokes/time_spoke.py:133
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly [b to region list, q to quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:33
msgid "Shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:48
#, fuzzy
msgid "Start shell"
msgstr "Iniciar o VNC"

#: pyanaconda/ui/tui/spokes/shell_spoke.py:57
msgid "Exit the shell to continue"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:54
#, fuzzy
msgid "Installation source"
msgstr "Progresso da Instalação"

#: pyanaconda/ui/tui/spokes/source.py:57 pyanaconda/ui/tui/spokes/source.py:106
#: pyanaconda/ui/gui/spokes/source.py:547
#: pyanaconda/ui/gui/spokes/source.glade:757
msgid "Closest mirror"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:98 pyanaconda/ui/gui/spokes/source.py:539
#, python-format
msgid "NFS server %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:100
#: pyanaconda/ui/gui/spokes/source.py:541
msgid "Local media"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:103
#: pyanaconda/ui/tui/spokes/source.py:117
#: pyanaconda/ui/gui/spokes/source.py:535
msgid "Error setting up software source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:108
#: pyanaconda/ui/tui/spokes/software.py:121
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:541
#: pyanaconda/ui/gui/spokes/software.py:224
#: pyanaconda/ui/gui/spokes/source.py:549
msgid "Nothing selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:119
#: pyanaconda/ui/tui/spokes/software.py:109
msgid "Processing..."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:135
msgid "CD/DVD"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:135
msgid "local ISO file"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:135
msgid "Network"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:139
#: pyanaconda/ui/gui/spokes/source.py:850
msgid ""
"The installation source is in use by the installer and cannot be changed."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:146
msgid "Choose an installation source type."
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:243
#: pyanaconda/ui/tui/spokes/source.py:281
msgid "Specify Repo Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:247
msgid "Repo URL"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:285
msgid "<server>:/<path>"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:286
msgid "NFS mount options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:328
#, fuzzy
msgid "Select device containing the ISO file"
msgstr "o dispositivo que contém o /boot"

#: pyanaconda/ui/tui/spokes/source.py:387
msgid "No mountable devices found"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:415
msgid "Select an ISO to use as install source"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:444
msgid "No *.iso files found in device root folder"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:40
#, fuzzy
msgid "Software selection"
msgstr "Desenvolvimento de Software"

#: pyanaconda/ui/tui/spokes/software.py:107
#: pyanaconda/ui/gui/spokes/software.py:208
msgid "Error checking software selection"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:111
#: pyanaconda/ui/gui/spokes/software.py:211
msgid "Installation source not set up"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:113
#: pyanaconda/ui/gui/spokes/software.py:214
msgid "Source changed - please verify"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:120
#: pyanaconda/ui/gui/spokes/software.py:222
msgid "Custom software selected"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:147
msgid "Installation source needs to be set up first."
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:157
msgid "Base environment"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:37
msgid "User creation"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:115 pyanaconda/ui/gui/spokes/user.py:362
msgid "No user will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:117
#, fuzzy
msgid "You must set a password"
msgstr "Definir senha de root"

#: pyanaconda/ui/tui/spokes/user.py:119 pyanaconda/ui/gui/spokes/user.py:364
#, python-format
msgid "Administrator %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:121 pyanaconda/ui/gui/spokes/user.py:366
#, python-format
msgid "User %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings.py:36
#, fuzzy
msgid "Warnings"
msgstr "Aviso"

#: pyanaconda/ui/tui/spokes/warnings.py:44
#: pyanaconda/ui/gui/spokes/welcome.glade:517
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: pyanaconda/ui/tui/spokes/progress.py:37
#, fuzzy
msgid "Progress"
msgstr "Em curso"

#: pyanaconda/ui/tui/spokes/progress.py:139
msgid "\tInstallation complete.  Press return to quit"
msgstr "\tInstalação completa.  Pressione Enter para sair"

#: pyanaconda/ui/tui/spokes/__init__.py:49
msgid "Default spoke title"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:61
msgid "testing status..."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:103
msgid "New value"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:128
#, python-format
msgid "%s: "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:129
#, fuzzy, python-format
msgid "%s (confirm): "
msgstr "Senha (confirmar): "

#: pyanaconda/ui/tui/spokes/__init__.py:132
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:154
msgid ""
"\n"
"Would you like to use it anyway?"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:171
msgid ""
"You have provided a password containing non-ASCII characters.\n"
"You may not be able to switch between keyboard layouts to login.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:177
#, python-format
msgid "Enter new value for '%s' and press enter\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:185
#, python-format
msgid ""
"You have provided an invalid user name: %s\n"
"Tip: Keep your user name shorter than 32 characters and do not use spaces.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:296
#, fuzzy
msgid "Password set."
msgstr "Senha está definida."

#: pyanaconda/ui/gui/__init__.py:334 widgets/src/BaseWindow.c:115
msgid "Help!"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:752 widgets/src/StandaloneWindow.c:51
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:767
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/__init__.py:767
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/hubs/__init__.py:224
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:111
msgid "Complete!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:211
#, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:214
msgctxt "GUI|Progress"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:222
#, python-format
msgid ""
"%s is now successfully installed, but some configuration still needs to be "
"done.\n"
"Finish it and then click the Finish configuration button please."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.py:227
#, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot to start using it!"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:107
#, python-format
msgid "Create a new %(container_type)s ..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:108
#, python-format
msgid "Create or select %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:110
msgid "Device reconfiguration failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:112
msgid ""
"Storage configuration reset due to unrecoverable error. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:143
#: pyanaconda/ui/gui/spokes/custom.glade:93
msgid "MANUAL PARTITIONING"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:369
#, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/custom.py:495
#, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:655
#, python-format
msgid ""
"/boot/efi must be on a device of type %(oneFsType)s or %(anotherFsType)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:659
#, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:662
#: pyanaconda/ui/gui/spokes/custom.py:664
#, python-format
msgid "%s cannot be encrypted"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:666
#, fuzzy
msgid "You must create a new file system on the root device."
msgstr "Tem de criar um novo sistema de ficheiros no dispositivo raiz."

#: pyanaconda/ui/gui/spokes/custom.py:671
#, python-format
msgid "Device does not support RAID level selection %s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:784
msgid "Device resize request failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:857
msgid "Device reformat request failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1228
#: pyanaconda/ui/gui/spokes/custom.py:2114
#, python-format
msgid "Invalid device name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1234
#, python-format
msgid "Specified name %s already in use."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1438
msgid "No disks assigned"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1444
#, python-format
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/custom.py:1502
msgid "The container is encrypted."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1538
msgid ""
"The space available to this mount point can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1540
msgid "This file system may not be resized."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1594
msgid ""
"Error checking storage configuration.  Click for details or press Done again "
"to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1596
msgid ""
"Warning checking storage configuration.  Click for details or press Done "
"again to continue."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1702
#, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1713
msgid "Failed to add new device. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1718
msgid "Invalid partition size set. Use a valid integer."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1825
msgid "Device removal request failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2055
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2085
#, python-format
msgid "(%s free)"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2232
#, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2238
#, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2321
msgid "No disks selected."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2326
msgid "Not enough free space on selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2331
#: pyanaconda/ui/gui/spokes/custom.py:2356
msgid "Automatic partitioning failed. Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2580
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2588
#, fuzzy
msgid "_Reset selections"
msgstr "Desenvolvimento de Software"

#: pyanaconda/ui/gui/spokes/custom.py:2588
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2663
msgid "Failed to unlock encrypted block device. Click for details"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:401
msgctxt "GUI|Spoke"
msgid "_TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:524
msgid "Restoring hardware time..."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:535
msgid "Invalid timezone"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:711
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:725
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:911
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:751
msgid "PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:713
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:722
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:910
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:913
msgid "AM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1061
msgid "You need to set up networking first if you want to use NTP"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1065
msgid "You have no working NTP server configured"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:451
msgctxt "GUI|Spoke"
msgid "_INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.py:573
#, python-format
msgctxt "GUI|Installation Destination|Filter"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/keyboard.py:48
#, python-format
msgid "%s to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:272
msgctxt "GUI|Spoke"
msgid "_KEYBOARD"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:359
msgid "Testing layouts configuration not available."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:367
msgid "Getting list of layouts..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.py:422
msgid "Layout switching not configured."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.py:54
msgctxt "GUI|Spoke"
msgid "_LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:87
msgid "Status unknown (missing)"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:90
msgid "Status unknown"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:92
msgid "Unmanaged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:95
#: pyanaconda/ui/gui/spokes/network.py:99
msgid "Unavailable"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:97
msgid "Firmware missing"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:103
msgid "Cable unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:105
msgid "Disconnected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:110
msgid "Connecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:112
msgid "Authentication required"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:114
msgid "Connected"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:116
msgid "Disconnecting"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:118
msgid "Connection failed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:303
msgid "Ethernet"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:304
msgid "Wireless"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:305
#: pyanaconda/ui/gui/spokes/network.glade:15
msgid "Bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:306
#: pyanaconda/ui/gui/spokes/network.glade:27
msgid "VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:307
#: pyanaconda/ui/gui/spokes/network.glade:23
msgid "Team"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:308
#: pyanaconda/ui/gui/spokes/network.glade:19
msgid "Bridge"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:729
msgid "unplugged"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:819
#: pyanaconda/ui/gui/spokes/network.glade:540
#: pyanaconda/ui/gui/spokes/network.glade:841
msgid "IPv4 Address"
msgstr "Endereço IPv4"

#: pyanaconda/ui/gui/spokes/network.py:820
#: pyanaconda/ui/gui/spokes/network.glade:516
#: pyanaconda/ui/gui/spokes/network.glade:853
#: pyanaconda/ui/gui/spokes/network.glade:1420
msgid "IPv6 Address"
msgstr "Endereço IPv6"

#: pyanaconda/ui/gui/spokes/network.py:822
#: pyanaconda/ui/gui/spokes/network.py:824
#: pyanaconda/ui/gui/spokes/network.glade:1408
msgid "IP Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:914
#, python-format
msgid "%d Mb/s"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1134
msgid "WEP"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1137
msgid "WPA"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1140
msgid "WPA2"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1144
msgid "Enterprise"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1149 widgets/src/SpokeSelector.c:59
#: pyanaconda/ui/gui/spokes/filter.glade:160
#: pyanaconda/ui/gui/spokes/filter.glade:566
#: pyanaconda/ui/gui/spokes/filter.glade:897
#: pyanaconda/ui/gui/spokes/filter.glade:1514
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:531
msgid "None"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1281
#: pyanaconda/ui/gui/spokes/network.glade:2435
msgid "Authentication required by wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1282
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1296
#, fuzzy
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/network.py:1304
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1314
#, fuzzy
msgid "User name: "
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/network.py:1325
msgid "Identity: "
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1330
#, fuzzy
msgid "Private key password: "
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/network.py:1386
msgctxt "GUI|Spoke"
msgid "_NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:52
msgctxt "GUI|Spoke"
msgid "_ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:99
#: pyanaconda/ui/gui/spokes/password.py:100
#, fuzzy
msgid "The password is set."
msgstr "Senha está definida."

#: pyanaconda/ui/gui/spokes/password.py:127
msgid "Root password is set"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:129
msgid "Root account is disabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:131
msgid "Root password is not set"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:52
msgctxt "GUI|Spoke"
msgid "_SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:98
#: pyanaconda/ui/gui/spokes/source.py:54
msgid "Downloading package metadata..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:101
msgid "Downloading group metadata..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:143
#: pyanaconda/ui/gui/spokes/source.py:529
msgid "Checking software dependencies..."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:148
msgid "Error checking software dependencies"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:374
msgid "Error checking software dependencies.  Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:478
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:483
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:484
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.py:485
#, fuzzy
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr "Desenvolvimento de Software"

#: pyanaconda/ui/gui/spokes/source.py:53
msgid "Setting up installation source..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:83
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:88
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:94
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:98
msgid "Proxy authentication data duplicated"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:231
msgid "This media is good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:233
msgid "This media is not good to install from."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:354
msgctxt "GUI|Spoke"
msgid "_INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:469
#: pyanaconda/ui/gui/spokes/source.py:673
msgid "Failed to set up installation source; check the repo url"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:533
#, fuzzy
msgid "Error setting up base repository"
msgstr "Erro ao Configurar o Repositório"

#: pyanaconda/ui/gui/spokes/source.py:544
#, fuzzy
msgid "Error setting up ISO file"
msgstr "Erro ao Configurar o Repositório"

#: pyanaconda/ui/gui/spokes/source.py:675
msgid ""
"Failed to set up installation source; check the repo url and proxy settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:704
#, python-format
msgid "Device: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:705
#, python-format
msgid "Label: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:907
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:911
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:928
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:931
msgid "NFS server is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:938
msgid "Invalid host name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:941
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:962
msgid "Duplicate repository names."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:969
msgid "Empty repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:972
msgid "Invalid repository name"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:978
msgid "Repository name conflicts with internal repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:117
msgid "Please wait... software metadata still loading."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:120
#, python-format
msgid ""
"Your current <b>%(product)s</b> software selection requires <b>%(total)s</b> "
"of available space, including <b>%(software)s</b> for software and <b>"
"%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:129
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including <b>"
"%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:179
#, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:189
#, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:213
#, python-format
msgid ""
"  You don't have enough space available to install <b>%(product)s</b>, even "
"if you used all of the free space available on the selected disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:225
#, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:246
msgctxt "GUI|Spoke"
msgid "INSTALLATION _DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:331
msgid "Saving storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:337
#: pyanaconda/ui/gui/spokes/storage.py:349
msgid "Failed to save storage configuration..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:385
#: pyanaconda/ui/gui/spokes/storage.py:682
#, fuzzy
msgid "Formatting DASDs"
msgstr "A formatar dispositivo DASD"

#: pyanaconda/ui/gui/spokes/storage.py:531
msgid "Error checking storage configuration.  Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:533
msgid "Warning checking storage configuration.  Click for details."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:574
#, python-format
msgid ""
"FCP device %(hba_id)s\n"
"WWPN %(wwpn)s\n"
"LUN %(lun)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:584
#, python-format
msgid "%s free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:631
#, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/storage.py:715
msgid ""
"You have chosen to skip boot loader installation.  Your system may not be "
"bootable."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:962
msgid "You'll be able to make space available during custom partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:978
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:983
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:984
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:998
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:1002
#, fuzzy
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/user.py:72
#, python-format
msgid "Invalid group name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:215
msgctxt "GUI|Spoke"
msgid "_USER CREATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:293 pyanaconda/ui/gui/spokes/user.py:294
msgid "The password was set by kickstart."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:329
msgid "Invalid user name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:331
msgid "Full name cannot contain colon characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.py:258
#, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:269
#, python-format
msgid ""
"The following nodes were discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the target IP address <b>%(targetAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:157
msgid "DATA"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:282
#, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:291
msgid "_Click here to create them automatically."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:316
msgid "Create new mount points by clicking the '+' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:324
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:329
msgid "_New mount points will use the following partitioning scheme:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:131
#, python-format
msgid ""
"<b>%(count)d disk; %(size)s capacity; %(free)s free space</b> (unpartitioned "
"and in file systems)"
msgid_plural ""
"<b>%(count)d disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in file systems)"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:190
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install boot loader"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:193
#: pyanaconda/ui/gui/spokes/lib/cart.glade:183
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:64
#, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:68
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:69
#, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
msgid "Volume Group"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:75
msgid "_Volume Group:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:76
msgid "Volume"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:76
msgid "_Volume:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:136
#, fuzzy
msgid "Cannot relabel already existing file system."
msgstr "O ponto de montagem %s deve estar num sistema de ficheiros linux."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:141
#, fuzzy
msgid "Cannot set label on file system."
msgstr "O ponto de montagem %s deve estar num sistema de ficheiros linux."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:143
#, fuzzy
msgid "Unacceptable label format for file system."
msgstr "O ponto de montagem %s deve estar num sistema de ficheiros linux."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:153
msgid "That mount point is already in use. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:155
msgid "Please enter a valid mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:157
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:169
msgid "That mount point is invalid. Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:310
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:389
#, fuzzy, python-format
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all other file systems in the %s root as well."
msgstr "Tem de criar um novo sistema de ficheiros no dispositivo raiz."

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:399
#, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:401
#, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:685
msgid "Invalid container name"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:49
#, fuzzy
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:59
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:35
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:36
msgid "Passphrases do not match."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:165
msgid "Passphrase contains non-ASCII characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:53
msgid "Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:54
msgid "Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:55
msgid "Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:114
msgid "This device contains the installation source."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:160
#, fuzzy, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr "A criar %(type)s em %(device)s"

#: pyanaconda/ui/gui/spokes/lib/resize.py:167
msgid "Not resizeable"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:191
msgid "Free space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:213
msgid ""
"You can remove existing file systems you no longer need to free up space for "
"this installation.  Removing a file system will permanently delete all of "
"the data it contains."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:219
msgid ""
"There is also free space available in pre-existing file systems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:229
#, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in file systems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in file systems)"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:236
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:451
#: pyanaconda/ui/gui/spokes/lib/resize.py:456
#: pyanaconda/ui/gui/spokes/lib/resize.glade:269
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:453
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:54
#, python-format
msgid "%(description)s (%(deviceName)s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.py:58
#, python-format
msgid "%(deviceName)s on %(container)s"
msgstr ""

#: widgets/src/BaseWindow.c:111
msgid "DISTRIBUTION INSTALLATION"
msgstr ""

#: widgets/src/BaseWindow.c:112
msgid "SPOKE NAME"
msgstr ""

#: widgets/src/BaseWindow.c:113
msgid "PRE-RELEASE / TESTING"
msgstr ""

#: widgets/src/DiskOverview.c:56
msgid "New Device"
msgstr ""

#: widgets/src/DiskOverview.c:58 widgets/src/DiskOverview.c:59
msgid "0 MB"
msgstr ""

#: widgets/src/MountpointSelector.c:54
msgid "0 GB"
msgstr ""

#: widgets/src/SpokeSelector.c:60
msgid "New Selector"
msgstr ""

#: widgets/src/SpokeWindow.c:48
msgid "_Done"
msgstr ""

#: widgets/src/StandaloneWindow.c:52
msgid "_Continue"
msgstr "_Continuar"

#: widgets/src/LayoutIndicator.c:34
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""

#: widgets/src/LayoutIndicator.c:35
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:36
#, fuzzy
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:50
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:81
msgid "ADVANCED USER CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:103
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:138
msgid "User and Group IDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:157
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:173
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:231
msgid "Group Membership"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:259
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:290
msgid "Example:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:307
msgid "wheel, my-team (1245), project-x (29935)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:341
msgid "Tip:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:360
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:110
#, fuzzy
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:124
#, fuzzy
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:149
msgid "Add and mark for usage NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:171
#, fuzzy
msgid "New NTP Server"
msgstr "Servidor Web"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:190
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:210
msgid "This URL refers to a pool of NTP servers"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:256
msgid "Pool"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:269
msgid "Working"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:277
msgid "Use"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:305
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:519
#, fuzzy
msgid "Configure NTP"
msgstr "Confirmar"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:326
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:934
msgid "TIME & DATE"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:376
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:406
msgid "Region"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:423
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:453
msgid "City"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:481
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr "Sem rede disponível"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:499
#, fuzzy
msgid "Use Network Time"
msgstr "Usar o modo de texto"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:586
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:599
msgid ":"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:619
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:637
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:655
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:673
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:691
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:735
msgid "AM/PM Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:771
msgid "AM/PM Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:822
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:840
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:899
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:60
#: pyanaconda/ui/gui/spokes/storage.glade:444
#: pyanaconda/ui/gui/spokes/storage.glade:1024
msgid "INSTALLATION DESTINATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:133
msgid "Search Results:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:146
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:161
msgid "Port / Target / LUN #"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:162
msgid "Target WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:199
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:224
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:252
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:293
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:366
#: pyanaconda/ui/gui/spokes/filter.glade:1267
#: pyanaconda/ui/gui/spokes/keyboard.glade:520
#: pyanaconda/ui/gui/spokes/source.glade:987
#: pyanaconda/ui/gui/spokes/lib/cart.glade:133
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:235
#: pyanaconda/ui/gui/spokes/lib/resize.glade:163
msgid "Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:380
#: pyanaconda/ui/gui/spokes/filter.glade:569
#: pyanaconda/ui/gui/spokes/filter.glade:748
msgid "WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:394
#: pyanaconda/ui/gui/spokes/filter.glade:762
#: pyanaconda/ui/gui/spokes/filter.glade:954
#: pyanaconda/ui/gui/spokes/filter.glade:1295
#: pyanaconda/ui/gui/spokes/lib/cart.glade:145
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:248
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:726
msgid "Capacity"
msgstr "Capacidade"

#: pyanaconda/ui/gui/spokes/filter.glade:408
#: pyanaconda/ui/gui/spokes/filter.glade:568
#: pyanaconda/ui/gui/spokes/filter.glade:791
#: pyanaconda/ui/gui/spokes/filter.glade:899
#: pyanaconda/ui/gui/spokes/filter.glade:983
msgid "Interconnect"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:422
msgid "Model"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:436
#: pyanaconda/ui/gui/spokes/filter.glade:1323
#: pyanaconda/ui/gui/spokes/filter.glade:1517
msgid "LUN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:450
#: pyanaconda/ui/gui/spokes/filter.glade:1337
msgid "Port"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:464
msgid "Target"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:478
#: pyanaconda/ui/gui/spokes/filter.glade:567
#: pyanaconda/ui/gui/spokes/filter.glade:776
#: pyanaconda/ui/gui/spokes/filter.glade:898
#: pyanaconda/ui/gui/spokes/filter.glade:968
msgid "Vendor"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:499
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:521
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:552
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:605
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:643
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:681
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:805
msgid "Paths"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:826
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:851
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:883
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:900
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:752
msgid "ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:940
msgid "Identifier"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1046
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1084
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1122
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1173
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1205
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "Filter B_y:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1215
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1253
#: pyanaconda/ui/gui/spokes/filter.glade:1515
msgid "CCW"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1281
#: pyanaconda/ui/gui/spokes/lib/summary.glade:155
msgid "Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1309
#: pyanaconda/ui/gui/spokes/filter.glade:1516
msgid "WWPN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1382
msgctxt "GUI|Installation Destination|Filter|zSeries|CCW"
msgid "_CCW:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1429
msgctxt "GUI|Installation Destination|Filter|zSeries|WWPN"
msgid "_WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1471
msgctxt "GUI|Installation Destination|Filter|zSeries|LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1541
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1564
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add zFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1579
msgctxt "GUI|Installation Destination|Filter"
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1594
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1609
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1624
msgid "Refresh _List"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:35
#, fuzzy
msgid "Add device"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/network.glade:49
#, fuzzy
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/network.glade:63
#, fuzzy
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr "_Adicionar"

#: pyanaconda/ui/gui/spokes/network.glade:92
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:165
#: pyanaconda/ui/gui/spokes/network.glade:2270
msgid "NETWORK & HOST NAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:204
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the host name here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:418
msgid "Slaves"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:442
#: pyanaconda/ui/gui/spokes/network.glade:878
#: pyanaconda/ui/gui/spokes/network.glade:1445
msgid "DNS"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:468
#: pyanaconda/ui/gui/spokes/network.glade:865
#: pyanaconda/ui/gui/spokes/network.glade:1432
msgid "Default Route"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:492
msgid "Subnet Mask"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:564
#: pyanaconda/ui/gui/spokes/network.glade:829
#: pyanaconda/ui/gui/spokes/network.glade:1276
msgid "Speed"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:588
#: pyanaconda/ui/gui/spokes/network.glade:817
msgid "Hardware Address"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:612
msgid "Parent"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:624
msgid "VLAN ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:699
#, fuzzy
msgctxt "GUI|Network|Wired"
msgid "_Configure..."
msgstr "Confirmar"

#: pyanaconda/ui/gui/spokes/network.glade:703
msgid "To apply the configuration immediately turn the device off and on."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:990
msgid "Security"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1002
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr "Sem rede disponível"

#: pyanaconda/ui/gui/spokes/network.glade:1016
msgid "Network Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1028
msgid "Security Key"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1111
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1128
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1145
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr "Confirmar"

#: pyanaconda/ui/gui/spokes/network.glade:1264
msgid "IMEI"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1288
msgid "Provider"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1470
#, fuzzy
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr "Confirmar"

#: pyanaconda/ui/gui/spokes/network.glade:1581
msgid "VPN Type"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1593
msgid "Gateway"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1605
msgid "Group Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1617
msgid "Group Password"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1629
#: pyanaconda/ui/gui/spokes/user.glade:116
#, fuzzy
msgid "User Name"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/network.glade:1736
#, fuzzy
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr "Confirmar"

#: pyanaconda/ui/gui/spokes/network.glade:1859
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1884
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1909
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1923
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1937
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:1951
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2140
msgid "Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2156
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2177
msgctxt "GUI|Network"
msgid "_Airplane Mode"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2196
msgid "More Network Config Box"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2216
#, fuzzy
msgctxt "GUI|Network"
msgid "_Host Name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/network.glade:2276
msgid "NETWORK CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2321
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2351
msgid "Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:2365
#, fuzzy
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/network.glade:2380
#, fuzzy
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr "Ligado!"

#: pyanaconda/ui/gui/spokes/network.glade:2454
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:10
msgid "SOFTWARE SELECTION"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:66
msgid "Base Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:84
msgid "Add-Ons for Selected Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:27
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Cancel & add more disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:43
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:78
#: pyanaconda/ui/gui/spokes/storage.glade:293
msgid "INSTALLATION OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:95
#: pyanaconda/ui/gui/spokes/storage.glade:309
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:121
#: pyanaconda/ui/gui/spokes/storage.glade:335
msgid "disk free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:137
#: pyanaconda/ui/gui/spokes/storage.glade:350
msgid "Free space available for use."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:153
#: pyanaconda/ui/gui/spokes/storage.glade:366
msgid "fs free"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:169
#: pyanaconda/ui/gui/spokes/storage.glade:382
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:192
#: pyanaconda/ui/gui/spokes/storage.glade:405
msgid "Here we'll describe what your options are."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:219
#, fuzzy
msgid "Need Space"
msgstr "Utilizar todo o espaço"

#: pyanaconda/ui/gui/spokes/storage.glade:242
#, fuzzy
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr "_Sair do instalador"

#: pyanaconda/ui/gui/spokes/storage.glade:258
#, fuzzy
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/storage.glade:432
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:507
msgid "Device Selection"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:526
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:544
msgid "Local Standard Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:608
#: pyanaconda/ui/gui/spokes/storage.glade:748
msgid "Disks left unselected here will not be touched."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:627
msgid "Specialized & Network Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:697
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:714
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:767
msgid "Other Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:793
#, fuzzy
msgid "Partitioning"
msgstr "Partição do Sistema EFI"

#: pyanaconda/ui/gui/spokes/storage.glade:810
#, fuzzy
msgid "Encryption"
msgstr "Cifrado"

#: pyanaconda/ui/gui/spokes/storage.glade:823
#, fuzzy
msgid "A_utomatically configure partitioning."
msgstr "Partição do Sistema EFI"

#: pyanaconda/ui/gui/spokes/storage.glade:839
msgid "_I will configure partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:871
msgid "_Encrypt my data."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:886
msgid "You'll set a passphrase next."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:907
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:923
msgid "Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:955
msgid "summary"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:988
msgctxt "GUI|Storage"
msgid "_Full disk summary and boot loader..."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:15
#: pyanaconda/ui/gui/spokes/keyboard.glade:388
msgid "KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:57
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:96
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:153
msgid "name"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:167
msgid "Selected Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:190
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:205
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:220
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:235
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:251
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:291
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:331
msgid "Alt + Shift to switch layouts."
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:346
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:421
#, fuzzy
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/keyboard.glade:435
#, fuzzy
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr "_Adicionar"

#: pyanaconda/ui/gui/spokes/keyboard.glade:467
msgid "ADD A KEYBOARD LAYOUT"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:484
msgid "You may add a keyboard layout by selecting it below:"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:533
msgid "Available Layouts"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:574
msgid "Add Layout"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:606
#, fuzzy
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/keyboard.glade:620
#, fuzzy
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/keyboard.glade:652
msgid "LAYOUT SWITCHING OPTIONS"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:669
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:705
msgid "use"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:718
msgid "description"
msgstr ""

#: pyanaconda/ui/gui/spokes/keyboard.glade:752
msgid "Layout Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:33
msgid "LANGUAGE SUPPORT"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:79
msgid "Select additional language support to be installed:"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:138
#: pyanaconda/ui/gui/spokes/langsupport.glade:241
#: pyanaconda/ui/gui/spokes/welcome.glade:273
#: pyanaconda/ui/gui/spokes/welcome.glade:372
msgid "nativeName"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:155
#: pyanaconda/ui/gui/spokes/welcome.glade:290
msgid "englishName"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:173
#: pyanaconda/ui/gui/spokes/welcome.glade:308
msgid "selected"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:193
#: pyanaconda/ui/gui/spokes/welcome.glade:336
msgid "Type here to search."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:227
msgid "checked"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:8
#: pyanaconda/ui/gui/spokes/password.glade:203
msgid "ROOT PASSWORD"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:54
#, fuzzy
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/password.glade:68
#, fuzzy
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr "Confirmar"

#: pyanaconda/ui/gui/spokes/password.glade:104
#: pyanaconda/ui/gui/spokes/user.glade:187
#, fuzzy
msgid "Confirm Password"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/password.glade:150
#: pyanaconda/ui/gui/spokes/user.glade:251
#, fuzzy
msgid "empty password"
msgstr "Definir senha de root"

#: pyanaconda/ui/gui/spokes/password.glade:171
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:29
#, fuzzy
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/source.glade:43
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:95
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:128
msgid "MEDIA VERIFICATION"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:155
msgid "Verifying media, please wait..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:224
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/source.glade:239
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/source.glade:267
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:300
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy URL:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:317
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:348
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:375
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User _name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/source.glade:392
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/source.glade:471
msgid "INSTALLATION SOURCE"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:530
msgid "Which installation source would you like to use?"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:544
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:571
msgid "Device:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:583
msgid "Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:592
#: pyanaconda/ui/gui/spokes/source.glade:692
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:616
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:644
#, fuzzy
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/source.glade:675
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:716
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:779
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:794
msgctxt "GUI|Software Source"
msgid "This URL refers to a _mirror list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:832
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:847
#: pyanaconda/ui/gui/spokes/source.glade:848
msgid "This field is optional."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:881
msgid "Updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:908
msgctxt "GUI|Software Source"
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the installation source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:933
msgid "Additional repositories"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:974
msgid "Enabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1019
#: pyanaconda/ui/gui/spokes/source.glade:1020
msgid "Add a new repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1021
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1038
#: pyanaconda/ui/gui/spokes/source.glade:1039
msgid "Remove the selected repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1040
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1057
#: pyanaconda/ui/gui/spokes/source.glade:1058
msgid "Revert to the previous list of repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1059
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1089
#: pyanaconda/ui/gui/spokes/source.glade:1090
msgid "URL for the repository, without protocol."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1102
#: pyanaconda/ui/gui/spokes/source.glade:1103
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1115
#: pyanaconda/ui/gui/spokes/source.glade:1116
#, fuzzy
msgid "Optional proxy user name."
msgstr "Erro operacional."

#: pyanaconda/ui/gui/spokes/source.glade:1128
#, fuzzy
msgid "Optional proxy password."
msgstr "Definir senha de root"

#: pyanaconda/ui/gui/spokes/source.glade:1140
msgctxt "GUI|Software Source"
msgid "This URL refers to a mirror _list."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1160
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/source.glade:1176
#: pyanaconda/ui/gui/spokes/source.glade:1177
msgid "Protocol for the repository URL."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1198
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1215
#, fuzzy
msgctxt "GUI|Software Source"
msgid "U_ser name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/source.glade:1232
#, fuzzy
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/source.glade:1248
#: pyanaconda/ui/gui/spokes/source.glade:1249
msgid "Name of the repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:23
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_Get me out of here!"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:37
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_I accept my fate."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:68
msgid "This is unstable, pre-release software."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:97
msgid ""
"You wake up inside an OS installer in Timbuktu, and it's six months in the "
"future.  But, there are bugs.  Bugs everywhere.  Bugs you must live with.  "
"This OS of the future isn't a stable OS you can rely on.  It's for testing "
"purposes only."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:126
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:225
msgid "What language would you like to use during the installation process?"
msgstr "Que língua gostaria de utilizar durante o processo de instalação?"

#: pyanaconda/ui/gui/spokes/welcome.glade:316
msgid "Languages"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:388
msgid "Locales"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:422
msgid "WELCOME"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:443
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:457
#, fuzzy
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr "_Continuar"

#: pyanaconda/ui/gui/spokes/welcome.glade:488
msgid "Unsupported Hardware Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:546
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:10
#: pyanaconda/ui/gui/spokes/user.glade:346
msgid "CREATE USER"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:62
msgctxt "GUI|User"
msgid "_Full name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:80
#, fuzzy
msgctxt "GUI|User"
msgid "_User name"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/user.glade:100
msgid "Full Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:131
#, fuzzy
msgctxt "GUI|User"
msgid "_Password"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/user.glade:149
#, fuzzy
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr "Definir senha de root"

#: pyanaconda/ui/gui/spokes/user.glade:201
msgid ""
"<b>Tip:</b> Keep your user name shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:211
msgctxt "GUI|User"
msgid "_Require a password to use this account"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:270
msgctxt "GUI|User"
msgid "_Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:289
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:62
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:539
msgid "RAID0 <span foreground=\"grey\">(Performance)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:66
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:543
msgid "RAID1 <span foreground=\"grey\">(Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:70
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:547
msgid "RAID4 <span foreground=\"grey\">(Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:551
msgid "RAID5 <span foreground=\"grey\">(Distributed Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:78
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:555
msgid "RAID6 <span foreground=\"grey\">(Redundant Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:82
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:559
msgid "RAID10 <span foreground=\"grey\">(Performance, Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:181
msgid "Add a new mount point."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:182
#, fuzzy
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr "_Adicionar"

#: pyanaconda/ui/gui/spokes/custom.glade:197
msgid "Remove the selected mount point(s)."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:198
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:213
msgid "Reload storage configuration from disk."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:214
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:279
#: pyanaconda/ui/gui/spokes/custom.glade:1013
#: pyanaconda/ui/gui/spokes/custom.glade:1157
#: pyanaconda/ui/gui/spokes/custom.glade:1244
msgid "Selected Device"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:297
#: pyanaconda/ui/gui/spokes/custom.glade:1031
#: pyanaconda/ui/gui/spokes/custom.glade:1175
#: pyanaconda/ui/gui/spokes/custom.glade:1262
msgid "Device description"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:358
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:405
#, fuzzy
msgid "_Desired Capacity:"
msgstr "Capacidade"

#: pyanaconda/ui/gui/spokes/custom.glade:456
msgid "_Label:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:503
#, fuzzy
msgid "Device(s):"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/custom.glade:533
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:565
msgid "_Name:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:620
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/custom.glade:664
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr "Cifrado"

#: pyanaconda/ui/gui/spokes/custom.glade:705
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:740
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:794
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:848
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:880
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/custom.glade:945
msgid "_Update Settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:968
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1065
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1086
#, fuzzy
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/custom.glade:1114
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1209
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1357
msgid "AVAILABLE SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1410
msgid "TOTAL SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1450
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:23
#, fuzzy
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:56
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:70
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:92
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:108
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _VLAN"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:144
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:187
msgid "Error adding FCoE SAN."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:215
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:40
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:54
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:86
msgid "ADD iSCSI STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:115
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:142
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr "Endereço IPv4"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:169
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr "Tem de indicar um nome para o iniciador iSCSI"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:185
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:751
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:186
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:752
msgid "CHAP pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:187
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:753
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:202
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:217
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:262
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _User name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:276
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:382
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _User name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:396
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:410
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User _name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:424
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:457
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:494
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:534
msgid "Discovery login rejected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:549
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:571
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:611
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:686
msgid "Node Name"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:697
msgid "Interface"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:708
msgid "Portal"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:737
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node Login Authentication Type:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:754
msgid "Use the credentials from discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:789
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _User name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:803
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:909
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _User name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:923
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:937
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP User name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:951
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:999
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1055
msgid "Node login failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1070
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1092
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:22
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:36
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:68
msgid "ADD DASD STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:97
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:125
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:145
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:182
msgid "Discovering DASD devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:224
#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:267
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:241
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:267
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:82
msgid "ADD zFCP STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:111
msgid ""
"To use zFCP disks, you must provide the device number, WWPN, and LUN "
"configured for the device."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:139
#, fuzzy
msgctxt "GUI|Advanced Storage|zFCP|Device Number"
msgid "_Device number:"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:168
msgid "WWPN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:188
msgctxt "GUI|Advanced Storage|zFCP|Start Discovery"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:225
msgid "Discovering zFCP devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:284
msgid ""
"The following error occurred discovering zFCP devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:310
msgctxt "GUI|Advanced Storage|zFCP|Retry Discovery"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/zfcp.glade:356
msgid "LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:26
#: pyanaconda/ui/gui/spokes/lib/cart.glade:78
msgid "SELECTED DISKS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:43
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:106
msgid "Boot"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:120
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:222
msgid "Description"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:158
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:261
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:739
msgid "Free"
msgstr "Livre"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:199
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:226
msgid "Disk summary goes here"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:21
#, fuzzy
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:37
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:129
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:144
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:176
msgid "CONFIGURE MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:191
msgid "Select one or more disks this device may reside on."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:321
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:336
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:369
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Desired Capacity:"
msgstr "Capacidade"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:386
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:404
msgid "ADD A NEW MOUNT POINT"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:421
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:456
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:480
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:535
msgid "Single <span foreground=\"grey\">(No Redundancy, No Striping)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:583
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:597
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:629
msgid "CONFIGURE CONTAINER"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:644
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:662
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr "Nome de utilizador"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:781
#, fuzzy
msgid "RAID Level:"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:809
#, fuzzy
msgid "Encrypt"
msgstr "Cifrado"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:850
msgid "Please enter a valid name."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:880
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:895
msgid "Automatic"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:896
msgid "As large as possible"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:897
msgid "Fixed"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:8
msgid "UNFORMATTED DASDS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:23
#, fuzzy
msgctxt "GUI|DAD Format Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:37
#, fuzzy
msgctxt "GUI|DAD Format Dialog"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:70
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs cannot be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:86
#, fuzzy
msgid "Unformatted DASDs Detected"
msgstr "A formatar dispositivo DASD"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:136
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:160
msgctxt "GUI|DAD Format Dialog"
msgid "_Format with dasdfmt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:207
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:247
msgid "Disk formatting complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:25
msgid "An unknown error occurred during installation.  Details are below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:23
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:38
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:66
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:82
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:103
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr "Senha: "

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:132
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr "Confirmar"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:270
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:8
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:71
msgid "RESCAN DISKS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:25
#, fuzzy
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:39
#, fuzzy
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:87
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:118
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:143
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:176
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:212
msgid "Scanning disks.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:223
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:254
msgid "Disk rescan complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:269
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:55
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr "_Cancelar"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:70
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:104
msgid "RECLAIM DISK SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:121
msgid "Description goes here."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:174
msgid "File System"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:185
msgid "Reclaimable Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:196
#: pyanaconda/ui/gui/spokes/lib/summary.glade:144
msgid "Action"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:221
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:237
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:253
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:367
msgid "Reclaim"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:25
#: pyanaconda/ui/gui/spokes/lib/summary.glade:89
msgid "SUMMARY OF CHANGES"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:42
#, fuzzy
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr "Partição do Sistema EFI"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:57
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:105
msgid ""
"Your customizations will result in the following changes taking effect after "
"you return to the main menu and begin installation:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:133
msgid "Order"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:166
#, fuzzy
msgid "Device"
msgstr "Dispositivo RAID"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:177
msgid "Mount point"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:35
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:53
msgid "Random data quality:"
msgstr ""

#: pyanaconda/ui/gui/main.glade:24
msgid "_Exit Installer"
msgstr ""

#: pyanaconda/ui/gui/main.glade:94
msgctxt "GUI|Main"
msgid "_No"
msgstr ""

#: pyanaconda/ui/gui/main.glade:108
msgctxt "GUI|Main"
msgid "_Yes"
msgstr ""

#: pyanaconda/ui/gui/main.glade:151
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""

#: pyanaconda/ui/gui/main.glade:175
msgid "Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:8
#: pyanaconda/ui/gui/hubs/summary.glade:147
msgid "INSTALLATION SUMMARY"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:84
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr ""

#: pyanaconda/ui/gui/hubs/summary.glade:100
#, fuzzy
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr "Pós Instalação"

#: pyanaconda/ui/gui/hubs/summary.glade:131
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:8
#: pyanaconda/ui/gui/hubs/progress.glade:261
msgid "CONFIGURATION"
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:91
msgid "Preparing to install"
msgstr "A preparar para instalar"

#: pyanaconda/ui/gui/hubs/progress.glade:149
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Finish configuration"
msgstr "A efectuar configuração pós-actualização"

#: pyanaconda/ui/gui/hubs/progress.glade:202
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Reboot"
msgstr "Re_iniciar"

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr "Instalar no Disco Rígido"

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr "Instalar"

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr "Instalar o Live CD no seu disco rígido"

#: /home/sbueno/scm/anaconda/data/liveinst/console.apps/liveinst:5
msgid "Starting Install to Hard Drive"
msgstr "A iniciar a instalação no disco rígido"

#: data/liveinst/gnome/fedora-welcome.desktop.in.h:1
#: data/liveinst/gnome/fedora-welcome:65
msgid "Welcome to Fedora"
msgstr ""

#: anaconda:463
#, fuzzy, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""
"%s requer %s MB de memória para instalar, mas você só tem %s MB nesta "
"máquina.\n"

#: anaconda:465
#, fuzzy, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""
"O %s instalador gráfico requer %s MB de memória, mas você só tem %s MB."

#: anaconda:468
msgid ""
"\n"
"Press <return> to reboot your system.\n"
msgstr ""

#: anaconda:470
msgid "Not enough RAM"
msgstr "RAM insuficiente"

#: anaconda:471
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""
" Tente o instalador em modo de texto executando:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
"a partir de um terminal de root."

#: anaconda:474
msgid " Starting text mode."
msgstr " A iniciar modo de texto."

#: anaconda:511
#, fuzzy
msgid "Fatal Error"
msgstr "Erro"

#: anaconda:636
msgid "DISPLAY variable not set. Starting text mode."
msgstr "A variável DISPLAY não está definida. A iniciar em modo de texto."

#: anaconda:644
msgid "Graphical installation is not available. Starting text mode."
msgstr "Instalação gráfica não disponível. A iniciar em modo de texto."

#: anaconda:652
#, fuzzy
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""
"O modo de texto disponibiliza um conjunto limitado de opções de instalação. "
"Não permite personalizar a disposição de partições nem a selecção de "
"pacotes. Em alternativa, deseja utilizar o modo VNC?"

#: anaconda:742
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr "Execute ssh install@%s para iniciar a instalação."

#: anaconda:744
msgid "Please ssh install@<host> to continue installation."
msgstr "Execute ssh install@<host> para continuar a instalação."

#: anaconda:985
msgid "Unable to create PID file"
msgstr ""

#: anaconda:986
#, python-format
msgid ""
"Anaconda is unable to create %s because the file already exists. Anaconda is "
"already running, or a previous instance of anaconda has crashed."
msgstr ""

#: data/liveinst/gnome/fedora-welcome:94
msgid "Try Fedora"
msgstr ""

#: data/liveinst/gnome/fedora-welcome:110
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""

#: data/liveinst/gnome/fedora-welcome:128
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""

#: data/liveinst/gnome/fedora-welcome:131
msgid "Close"
msgstr ""

#~ msgid ""
#~ "Your system had dirty file systems which you chose not to mount.  Press "
#~ "return to get a shell from which you can fsck and mount your partitions. "
#~ "%s"
#~ msgstr ""
#~ "O sistema tem alguns sistemas de ficheiros com erros que você escolheu "
#~ "não montar.  Pressione Enter para obter uma consola de onde pode executar "
#~ "o fsck e montar as partições. %s"

#~ msgid "_Fedora"
#~ msgstr "_Fedora"

#~ msgid "Red Hat Enterprise Linux"
#~ msgstr "Red Hat Enterprise Linux"

#~ msgid "c"
#~ msgstr "c"

#, fuzzy
#~ msgctxt "GUI|Network|Wired"
#~ msgid "C_onfigure..."
#~ msgstr "Confirmar"

#, fuzzy
#~ msgctxt "GUI|Software Source|Proxy Dialog"
#~ msgid "_Ok"
#~ msgstr "OK"

#, fuzzy
#~ msgctxt "GUI|Software Source|Proxy Dialog"
#~ msgid "Pass_word"
#~ msgstr "Senha: "

#~ msgid ""
#~ "Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or "
#~ "'0-9'"
#~ msgstr ""
#~ "O nome da máquina deve começar com um carácter válido no intervalo 'a-z', "
#~ "'A-Z' ou '0-9'"

#~ msgid "You need more space on the following file systems:\n"
#~ msgstr "É necessário mais espaço nos seguintes sistemas de ficheiros:\n"

#~ msgid "Can't have a question in command line mode!"
#~ msgstr "Não pode haver questões no modo de linha de comandos!"

#, fuzzy
#~ msgid "<b>User_name</b>"
#~ msgstr "Nome de utilizador"

#, fuzzy
#~ msgid "<b>Pass_word</b>"
#~ msgstr "Senha: "

#, fuzzy
#~ msgid "R_AID Level:"
#~ msgstr "Dispositivo RAID"

#~ msgid "Completed"
#~ msgstr "Concluído"

#~ msgid ""
#~ "(%s)\n"
#~ "Command line mode requires all choices to be specified in a kickstart "
#~ "configuration file."
#~ msgstr ""
#~ "(%s)\n"
#~ "O modo de comando de linha requer que todas as escolhas sejam "
#~ "especificadas num ficheiro de configuração de arranque."

#~ msgid ""
#~ "The following error was found while parsing the kickstart configuration "
#~ "file:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "Foi encontrado o seguinte erro ao processar o ficheiro de configuração "
#~ "kickstart:\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "(%s)\n"
#~ "%s"
#~ msgstr ""
#~ "(%s)\n"
#~ "%s"

#~ msgid "In interactive step can't continue. (%s)"
#~ msgstr "Não pode continuar em passo interetivo. (%s)"

#~ msgid ""
#~ "An unhandled exception has occurred.  This is most likely a bug.  Please "
#~ "save a copy of the detailed exception and file a bug report"
#~ msgstr ""
#~ "Ocorreu uma excepção não tratada.  Isto é provavelmente um erro.  Guarde "
#~ "uma cópia detalhada da excepção e submeta um relatório de erro"

#~ msgid " with the provider of this software."
#~ msgstr " com o fornecedor deste software."

#~ msgid " against anaconda at %s"
#~ msgstr " com o anaconda em %s"

#~ msgid "Install on System"
#~ msgstr "Instalar no sistema"

#~ msgid "Exit installer"
#~ msgstr "Sair do instalador"

#~ msgid ""
#~ "After shutdown, please perform a manual IPL from %s to continue "
#~ "installation."
#~ msgstr ""
#~ "Depois de desligar, por favor, efectue um IPL manual a partie de %s para "
#~ "continuar a instalação."

#~ msgid "IP address is missing."
#~ msgstr "Falta o endereço IP."

#~ msgid ""
#~ "IPv4 addresses must contain four numbers between 0 and 255, separated by "
#~ "periods."
#~ msgstr ""
#~ "Os endereços IPv4 devem conter quatro números entre 0 e 255, separados "
#~ "por pontos."

#~ msgid "'%s' is not a valid IPv6 address."
#~ msgstr "'%s' não é um endereço IPv6 válido."

#~ msgid "'%s' is an invalid IP address."
#~ msgstr "'%s' não é um endereço IP válido."

#~ msgid "bootloader device"
#~ msgstr "dispositivo de arranque"

#~ msgid "First sector of boot partition"
#~ msgstr "Primeiro sector da partição de arranque"

#~ msgid "Master Boot Record"
#~ msgstr "Master Boot Record"

#~ msgid "Apple EFI Boot Partition"
#~ msgstr "Partição de arranque Apple EFI"

#~ msgid "PReP Boot Partition"
#~ msgstr "Partição PReP Boot"

#~ msgid "Apple Bootstrap Partition"
#~ msgstr "Partição Apple Bootstrap"

#~ msgid "DASD"
#~ msgstr "DASD"

#~ msgid "zFCP"
#~ msgstr "zFCP"

#~ msgid "%s MB"
#~ msgstr "%s MB"

#~ msgid "%s KB"
#~ msgstr "%s KB"

#~ msgid "%s Byte"
#~ msgid_plural "%s Bytes"
#~ msgstr[0] "%s Byte"
#~ msgstr[1] "%s Bytes"

#~ msgid "<b>Upgrading %(pkgStr)s</b> (%(size)s)\n"
#~ msgstr "<b>A actualizar %(pkgStr)s</b> (%(size)s)\n"

#~ msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
#~ msgstr "<b>A instalar %(pkgStr)s</b> (%(size)s)\n"

#~ msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
#~ msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
#~ msgstr[0] "Pacote completo: %(donepkgs)d of %(numpkgs)d"
#~ msgstr[1] "Pacotes completos: %(donepkgs)d of %(numpkgs)d"

#~ msgid "<b>Cleaning up %s</b>"
#~ msgstr "<b>A limpar %s</b>"

#~ msgid "cpio"
#~ msgstr "cpio"

#~ msgid "unpack"
#~ msgstr "desempacotar"

#~ msgid "script"
#~ msgstr "script"

#~ msgid ""
#~ "A %s error occurred when installing the %s package.  This could indicate "
#~ "errors when reading the installation media.  Installation cannot continue."
#~ msgstr ""
#~ "Um erro %s ocorreu durante a instalação do pacote %s.  Isto pode indicar "
#~ "erros durante a leitura do mídia de instalação. A instalação não pode "
#~ "continuar."

#~ msgid "Wrong Disc"
#~ msgstr "Disco Errado"

#~ msgid "That's not the correct %s disc."
#~ msgstr "Este não é o disco %s correcto."

#~ msgid "Repository %r is missing name in configuration, using id"
#~ msgstr "O repositório %r não tem nome na configuração; a usar o id"

#~ msgid ""
#~ "Some of your software repositories require networking, but there was an "
#~ "error enabling the network on your system."
#~ msgstr ""
#~ "Alguns dos repositórios de software necessitam de rede, mas houve um erro "
#~ "ao activar a rede no seu sistema."

#~ msgid "_Eject"
#~ msgstr "_Ejectar"

#~ msgid "_Retry"
#~ msgstr "_Repetir"

#~ msgid ""
#~ "The file %s cannot be opened.  This is due to a missing file, a corrupt "
#~ "package or corrupt media.  Please verify your installation source.\n"
#~ "\n"
#~ "If you exit, your system will be left in an inconsistent state that will "
#~ "likely require reinstallation.\n"
#~ "\n"
#~ msgstr ""
#~ "Não foi possível abrir o ficheiro %s.  Isto deve-se a um ficheiro em "
#~ "falta ou talvez um pacote estragado.  Verifique o seu suporte físico de "
#~ "instalação.\n"
#~ "\n"
#~ "Se sair, o seu sistema ficará num estado inconsistente que poderá "
#~ "necessitar de reinstalação.\n"
#~ "\n"

#~ msgid "Retrying"
#~ msgstr "A repetir"

#~ msgid "Retrying download."
#~ msgstr "A repetir a transferência."

#~ msgid ""
#~ "There was an error running your transaction for the following reason: %s\n"
#~ msgstr ""
#~ "Ocorreu um erro ao executar a sua transacção, pela seguinte razão: %s\n"

#~ msgid "_Back"
#~ msgstr "_Anterior"

#~ msgid ""
#~ "There was an error running your transaction for the following reason: "
#~ "%s.\n"
#~ msgstr ""
#~ "Ocorreu um erro ao executar a transacção pelo seguinte motivo: %s.\n"

#~ msgid "Error Running Transaction"
#~ msgstr "Erro ao Executar a Transacção"

#~ msgid "file conflicts"
#~ msgstr "conflitos de ficheiros"

#~ msgid "older package(s)"
#~ msgstr "pacote(s) antigo(s)"

#~ msgid "insufficient disk space"
#~ msgstr "espaço insuficiente em disco"

#~ msgid "insufficient disk inodes"
#~ msgstr "i-nodes do disco insuficientes"

#~ msgid "package conflicts"
#~ msgstr "conflitos de pacotes"

#~ msgid "package already installed"
#~ msgstr "o pacote já está instalado"

#~ msgid "required package"
#~ msgstr "pacote obrigatório"

#~ msgid "package for incorrect arch"
#~ msgstr "o pacote é para uma arquitectura incorrecta"

#~ msgid "package for incorrect os"
#~ msgstr "pacote para um sistema operativo incorrecto"

#~ msgid ""
#~ "There were file conflicts when checking the packages to be installed:\n"
#~ "%s\n"
#~ msgstr ""
#~ "Ocorreram conflitos de ficheiros ao verificar os pacotes a instalar:\n"
#~ "%s\n"

#~ msgid ""
#~ "There was an error running your transaction for the following reason(s): "
#~ "%s.\n"
#~ msgstr ""
#~ "Ocorreu um erro ao executar a transacção pelas seguintes razões: %s.\n"

#~ msgid ""
#~ "Unable to read group information from repositories.  This is a problem "
#~ "with the generation of your install tree."
#~ msgstr ""
#~ "Não é possível ler a informação dos grupos dos repositórios.  Este é um "
#~ "problema com a geração da sua árvore de instalação."

#~ msgid "Edit"
#~ msgstr "Editar"

#~ msgid ""
#~ "Unable to read package metadata. This may be due to a missing repodata "
#~ "directory.  Please ensure that your install tree has been correctly "
#~ "generated.\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "Não foi possível ler os meta-dados do pacote. Isto poderá ser devido a "
#~ "uma pasta repodata em falta.  Verifique que a sua árvore de instalação "
#~ "foi gerada correctamente.\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "Some of the packages you have selected for install are missing "
#~ "dependencies.  You can exit the installation, go back and change your "
#~ "package selections, or continue installing these packages without their "
#~ "dependencies.  If you continue, these packages may not work correctly due "
#~ "to missing components."
#~ msgstr ""
#~ "Alguns dos pacotes que seleccionou para instalar têm dependências em "
#~ "falta.  Pode sair da instalação, voltar atrás e alterar a sua selecção de "
#~ "pacotes, ou continuar a instalação destes pacotes sem as suas "
#~ "dependências.  Se continuar, estes pacotes podem não funcionar "
#~ "correctamente devido aos componentes em falta."

#~ msgid ""
#~ "Your selected packages require %d MB of free space for installation, but "
#~ "you do not have enough available.  You can change your selections or exit "
#~ "the installer."
#~ msgstr ""
#~ "Os pacotes que seleccionou necessitam de %d MB em espaço livre para a "
#~ "instalação, mas você não tem espaço disponível suficiente.  Poderá mudar "
#~ "a sua selecção ou sair do instalador."

#~ msgid "Post Upgrade"
#~ msgstr "Actualização Posterior"

#~ msgid "Performing post-installation configuration"
#~ msgstr "A efectuar configuração pós-instalação"

#~ msgid "Starting installation process"
#~ msgstr "A iniciar processo de instalação"

#~ msgid "Dependency Check"
#~ msgstr "Verificação de Dependências"

#~ msgid "Checking dependencies in packages selected for installation"
#~ msgstr "A verificar dependências dos pacotes seleccionados para instalação"

#~ msgid "Retrieving installation information."
#~ msgstr "A obter informação de instalação."

#~ msgid "Retrieving installation information for %s."
#~ msgstr "A obter informação de instalação para %s."

#~ msgid ""
#~ "The default installation of %s includes a set of software applicable for "
#~ "general internet usage. You can optionally select a different set of "
#~ "software now."
#~ msgstr ""
#~ "A instalação por omissão de %s inclui um conjunto de aplicações genéricas "
#~ "destinadas à utilização de Internet. Opcionalmente, pode seleccionar "
#~ "agora um conjunto diferente de software."

#~ msgid "Graphical Desktop"
#~ msgstr "Desktop Gráfico"

#~ msgid "Minimal"
#~ msgstr "Mínima"

#~ msgid ""
#~ "The default installation of %s is a minimum install. You can optionally "
#~ "select a different set of software now."
#~ msgstr ""
#~ "A instalação por omissão do %s é uma instalação mínima. Pode "
#~ "opcionalmente seleccionar agora um conjunto de software diferente."

#~ msgid "Upgrading"
#~ msgstr "A actualizar"

#~ msgid "This partition is holding the data for the hard drive install."
#~ msgstr ""
#~ "Esta partição contém os dados para a instalação a partir do disco rígido."

#~ msgid "You cannot delete a partition of a LDL formatted DASD."
#~ msgstr "Não pode apagar uma partição de um DASD formatado com LDL."

#~ msgid "This device is part of the RAID device %s."
#~ msgstr "Esta partição faz parte do dispositivo RAID %s."

#~ msgid "This device is part of a RAID device."
#~ msgstr "Esta partição faz parte de um dispositivo RAID."

#~ msgid "This device is part of an inconsistent LVM Volume Group."
#~ msgstr "Este dispositivo é parte de um Grupo de Volume LVM inconsistente."

#~ msgid "This device is part of the LVM volume group '%s'."
#~ msgstr "Esta partição faz parte do grupo de volumes LVM '%s'."

#~ msgid "This device is part of a LVM volume group."
#~ msgstr "Esta partição faz parte de um grupo de volumes LVM."

#~ msgid ""
#~ "This device is an extended partition which contains logical partitions "
#~ "that cannot be deleted:\n"
#~ "\n"
#~ msgstr ""
#~ "Este dispositivo é uma partição estendida que contêm partições lógicas "
#~ "que não podem ser removidas:\n"
#~ "\n"

#~ msgid ""
#~ "Installing on a USB device.  This may or may not produce a working system."
#~ msgstr ""
#~ "A instalar num dispositivo USB.  Isto pode criar um sistema funcional ou "
#~ "não."

#~ msgid ""
#~ "Installing on a FireWire device.  This may or may not produce a working "
#~ "system."
#~ msgstr ""
#~ "A instalar num dispositivo FireWire.  Isto pode criar um sistema "
#~ "funcional ou não."

#~ msgid "you have not created a bootloader stage1 target device"
#~ msgstr "não criou um dispositivo de arranque stage1 de destino"

#~ msgid "Preparing %d DASD device for use with Linux..."
#~ msgid_plural "Preparing %d DASD devices for use with Linux..."
#~ msgstr[0] "A preparar %d dispositivo DASD para utilização com Linux..."
#~ msgstr[1] "A preparar %d dispositivos DASD para utilização com Linux..."

#~ msgid "Resizing filesystem on %(device)s"
#~ msgstr "A redimensionar o sistema de ficheiros em %(device)s"

#~ msgid "Migrating filesystem on %(device)s"
#~ msgstr "A migrar sistema de ficheiros em %(device)s"

#~ msgid "vginfo failed for %s"
#~ msgstr "vginfo falhou para %s"

#~ msgid "lvs failed for %s"
#~ msgstr "lvs falhou para %s"

#~ msgid "A RAID%(raidLevel)d set requires at least %(minMembers)d member"
#~ msgid_plural ""
#~ "A RAID%(raidLevel)d set requires at least %(minMembers)d members"
#~ msgstr[0] "Um RAID%(raidLevel)d requer pelo menos %(minMembers)d membro"
#~ msgstr[1] "Um RAID%(raidLevel)d requer pelo menos %(minMembers)d membros"

#~ msgid "Back"
#~ msgstr "Anterior"

#~ msgid ""
#~ "Are you sure you want to skip entering a passphrase for device %s?\n"
#~ "\n"
#~ "If you skip this step the device's contents will not be available during "
#~ "installation."
#~ msgstr ""
#~ "Tem a certeza que não deseja introduzir uma frase-senha para o "
#~ "dispositivo %s?\n"
#~ "\n"
#~ "Se ignorar este passo, o conteúdo do dispositivo ficará indisponível "
#~ "durante a instalação."

#~ msgid "FCoE not available"
#~ msgstr "FCoE não disponível"

#~ msgid "Unknown return code: %d."
#~ msgstr "Código de retorno desconhecido: %d."

#~ msgid "%(type)s filesystem check failure on %(device)s: "
#~ msgstr ""
#~ "A verificação do sistema de ficheiros %(type)s em %(device)s falhou: "

#~ msgid "File system errors left uncorrected."
#~ msgstr "Erros no sistema de ficheiros deixados por corrigir."

#~ msgid "Usage or syntax error."
#~ msgstr "Erro de utilização ou de sintaxe."

#~ msgid "e2fsck cancelled by user request."
#~ msgstr "e2fsck cancelado a pedido do utilizador."

#~ msgid "Shared library error."
#~ msgstr "Erro de biblioteca partilhada."

#~ msgid ""
#~ "Recoverable errors have been detected or dosfsck has discovered an "
#~ "internal inconsistency."
#~ msgstr ""
#~ "Erros recuperáveis foram detectados ou o dosfsck descobriu uma "
#~ "inconsistência interna."

#~ msgid "Usage error."
#~ msgstr "Erro de utilização."

#~ msgid "Unable to change iSCSI initiator name once set"
#~ msgstr "Incapaz de alterar o nome do iniciador iSCSI uma vez definido"

#~ msgid "iSCSI not available"
#~ msgstr "iSCSI não disponível"

#~ msgid "No initiator name set"
#~ msgstr "Nome do iniciador não configurado"

#~ msgid "Logging in to iSCSI node"
#~ msgstr "A iniciar sessão no nó iSCSI"

#~ msgid "Logging in to iSCSI node %s"
#~ msgstr "A iniciar sessão no nó iSCSI %s"

#~ msgid "No iSCSI nodes discovered"
#~ msgstr "Não foram encontrados nós iSCSI"

#~ msgid "No new iSCSI nodes discovered"
#~ msgstr "Não foram encontrados novos nós iSCCI"

#~ msgid "Could not log in to any of the discovered nodes"
#~ msgstr "Não foi possível iniciar sessão em nenhum dos nós descobertos"

#~ msgid "You have not specified a device number or the number is invalid"
#~ msgstr "Não indicou um número de dispositivo ou o número é inválido"

#~ msgid "You have not specified a worldwide port name or the name is invalid."
#~ msgstr ""
#~ "Não indicou um nome de porto mundialmente global ou o nome é inválido."

#~ msgid "You have not specified a FCP LUN or the number is invalid."
#~ msgstr "Não indicou um LUN FCP ou o número é inválido."

#~ msgid "zFCP device %s not found, not even in device ignore list."
#~ msgstr ""
#~ "O dispositivo zFCP %s não foi encontrado, nem mesmo na lista de "
#~ "dispositivos a ignorar."

#~ msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
#~ msgstr ""
#~ "Não foi possível definir o dispositivo zFCP %(devnum)s online (%(e)s)."

#~ msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Não foi possível adicionar WWPN %(wwpn)s ao dispositivo zFCP %(devnum)s "
#~ "(%(e)s)."

#~ msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
#~ msgstr "WWPN %(wwpn)s não encontrado no dispositivo zFCP %(devnum)s."

#~ msgid ""
#~ "Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s "
#~ "(%(e)s)."
#~ msgstr ""
#~ "Não foi possível adicionar LUN %(fcplun)s ao WWPN %(wwpn)s no dispositivo "
#~ "zFCP %(devnum)s (%(e)s)."

#~ msgid ""
#~ "LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already "
#~ "configured."
#~ msgstr ""
#~ "LUN %(fcplun)s em WWPN %(wwpn)s no dispositivo zFCP %(devnum)s já se "
#~ "encontra configurada."

#~ msgid ""
#~ "Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on "
#~ "zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Não foi possível ler atributo da LUN %(fcplun)s em falha em WWPN %(wwpn)s "
#~ "no dispositivo zFCP %(devnum)s (%(e)s)."

#~ msgid ""
#~ "Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
#~ "again."
#~ msgstr ""
#~ "LUN %(fcplun)s em falha em WWPN %(wwpn)s no dispositivo zFCP %(devnum)s "
#~ "removida novamente."

#~ msgid ""
#~ "Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s "
#~ "%(fcplun)s (%(e)s)."
#~ msgstr ""
#~ "Não foi possível remover correctamente o dispositivo SCSI do zFCP "
#~ "%(devnum)s %(wwpn)s %(fcplun)s (%(e)s)."

#~ msgid ""
#~ "Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device "
#~ "%(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Não foi possível remover LUN %(fcplun)s ao WWPN %(wwpn)s no dispositivo "
#~ "zFCP %(devnum)s (%(e)s)."

#~ msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "Não foi possível remover WWPN %(wwpn)s no dispositivo zFCP %(devnum)s "
#~ "(%(e)s)."

#~ msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
#~ msgstr ""
#~ "Não foi possível definir o dispositivo zFCP %(devnum)s offline (%(e)s)."
